### PR TITLE
Hit the api

### PR DESCRIPTION
**Pull request by PierreBarba**
# hello reviewer ✋🏽 😄 welcome to my repo thanks for the support have a great day

## Description

> In this activity you will continue working on the Leaderboard list app, sending and receiving data from/to the Leaderboard API.

## General requirements

> Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
> Make sure that you used correct flow [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
> Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

## JavaScript requirements

> Follow our list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).

## Project requirements

> Read the [Leaderboard API documentation](https://www.notion.so/microverse/Leaderboard-API-service-24c0c3c116974ac49488d4eb0267ade3) to learn how to use this API.
> Create a new game with the name of your choice by using the API.
> Make sure that you saved the ID of your game that will be returned by API.
> Implement the "Refresh" button that will get all scores for a game created by you from the API (receiving data from the API and parsing the JSON).
> Implement the form "Submit" button that will save a score for a game created by you (sending data to the API).
> Use arrow functions instead of the function keyword.
> Use async and await JavaScript features to consume the API.
> No styling is required. 

## workflow

- delete my display file
- add addScore file
- add getScore file
- get my api identifier
- use my identifier ID
- I consume the API and use it in my index to give functionality to the application
- fix my app details
- fix linter bugs

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,26 +1,31 @@
...
-    this.name = name;
-    this.score = score;
+const displayScores = async () => {
+  const data = await getScores();
+  sectionForScores.innerHTML = '';
+  for (let i = data.result.length - 1; i >= 0; i -= 1) {
+    const li = document.createElement('li');
+    li.classList.add('score');
+    li.innerHTML = `${data.result[i].user}: ${data.result[i].score}`;
+    sectionForScores.appendChild(li);
```
Comment (OLIPLICHE) [inline src/index.js:14]: - Okay 👍 
- 
Capture

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @PierreBarba, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linter is green (the last commit have a green check mark (✅)) 👏
- Commit messages are meaningful 👏 👏
- Gitflow is correctly used (Pull Request is made from the feature branch into the dev branch) 👏

## Required Changes ♻️

- [x]  Kindly restructure your Readme: The different sections you mentioned in the template must be the same inside the Table of contents. 

- [x] The sections need to be meaningful: 

Capture


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (PierreBarba) [reply src/index.js:14]: thanks for the support, have a great day 😄

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @PierreBarba, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights:
- All linters passing
- All requirements completed
- Professional readme

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**